persistence/src/main/java/edu/unc/lib/dl/services/OperationsMessageSender.java

```diff
@@ -314,6 +314,31 @@ public String sendEditTypeOperation(String userid, Collection<PID> pids, Resourc
         return getMessageId(msg);
     }
 
+    /**
+     * Sends Update MODS operation message.
+     *
+     * @param userid id of user who triggered the operation
+     * @param pids objects whose MODS changed
+     * @return id of operation message
+     */
+    public String sendUpdateDescriptionOperation(String userid, Collection<PID> pids) {
+        Element contentEl = createAtomEntry(userid, pids.iterator().next(),
+                CDRActions.UPDATE_MODS);
```

Comment:
Mind if we make this action UPDATE_DESCRIPTION to match most of the rest of the naming, since this is effectively api for external consumption?

Reply:
done!